cfme/tests/automate/test_automate_manual.py

```diff
@@ -930,3 +930,127 @@ def test_git_refresh_with_renamed_yaml():
         1716443
     """
     pass
+
+
+@pytest.mark.tier(2)
+def test_upload_blank_line():
+    """
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/8h
+        caseposneg: negative
+        startsin: 5.10
+        casecomponent: Automate
+        testSteps:
+            1. Create blank zip(test.zip) and yaml(test.yml) file
+            2. Navigate to Automation > Automate > Import/Export and upload test.zip file
+            3. Navigate to Automation > Automate > Customization > Import/Export and upload test.yml
+        expectedResults:
+            1.
+            2. Error message should be displayed
+            3. Error message should be displayed
+
+    Bugzilla:
+        1720611
+    """
+    pass
+
+
+@pytest.mark.tier(1)
+def test_initiate_retirement_automate_method():
+    """
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/8h
+        caseposneg: positive
+        startsin: 5.10
+        casecomponent: Automate
+        testSteps:
+            1. Create a writeable domain
+            2. Create a /System/Request/retire_automation_service instance and set meth5 to
+               retire_automation_service.
+            3. Create a /System/Request/retire_automation_service method.
+               Sample code:
+               ```
+               service = $evm.root['service']
+               $evm.log(:info, "create_retire_request for  service #{service}")
+               request = $evm.execute(:create_retire_request, service)
+               $evm.log(:info, "Create request for create_retire_request for request #{request}")
+               ```
+            4. In Automation/Automate/Simulation:
+               >> Set Request => retire_automation_service
+               >> Select Object Attribute Type => Service
+               >> Selection => Service to be retired
+               and click on Submit
+        expectedResults:
+            1.
+            2.
+            3.
+            4. Service should be retired from Automate method.
+
+    Bugzilla:
+        1700524
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+def test_git_refresh_with_rapid_updates():
+    """
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/8h
+        caseposneg: positive
```

Comment:
Isn't this a negative test case?

Reply:
- Yeah there is confusion in mind about it.
- Because if you see in this test case first we are checking if broking domain is imported or not(here we want error should be displayed) and at the end we are fixing this broken domain to import it successfully(here we want that fixed domain to import successfully).
- Hence keeping it as positive.

Reply:
Yeah, it's kind of a mix of positive and negative. I agree that the end result is positive, so positive makes the most sense.